platforms/hyperledger-indy/configuration/roles/create/helm_component/ledger_txn/tasks/identity.yaml

```diff
@@ -0,0 +1,19 @@
+#----------------------------------------------------------------------  
+    # create deployment file for node steward
+    - name: "Create new identity ledger transaction deployment file"
+      include_role:
+        name: ledger-tpl
```

Comment:
What's this role please?

Reply:
This role is used for running the helm release role .(as a starting point)

Reply:
And where is it located in the Git repo?